css/selectors/closed.json

```diff
@@ -33,9 +32,9 @@
             "webview_ios": "mirror"
           },
           "status": {
-            "experimental": true,
-            "standard_track": true,
-            "deprecated": false
+            "experimental": false,
+            "standard_track": false,
```

Comment:
I don't think we change `standard_track` when things are removed from spec, that's what we have `deprecated` for.

```suggestion
            "standard_track": true,
```

Reply:
We should probably align, and document this in https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#choosing-status-properties.